templates/docs/examples/patterns/lists/lists-dividers-dotted.html

```diff
@@ -0,0 +1,18 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Lists / Dotted with horizontal divider{% endblock %}
+
+{% block standalone_css %}patterns_lists{% endblock %}
+
+{% block content %}
+<ul class="p-list--divided">
+  <li class="p-list__item is-dotted">Balance compute load in the cloud</li>
+  <li class="p-list__item nested-list-wrapper">
```

Comment:
Here I added a class to specify that this list element is just a wrapper for the nested list and shouldn't have a top border or a dot